api/src/Controller/BaseController.php

```diff
@@ -179,7 +122,7 @@ abstract class BaseController
     /**
      * @var array[]
     */
-    protected $includes;
+    public $includes;
```

Comment:
This is because the new `IncludeResource` class need to access this int he base class and it is no longer in the same module so it no longer has access to protected members.  There is no friendship in PHP to allow unrelated classed to access protected members.

Reply:
OK. I'll let this one slide since it's step 1 of moving `IncludeResource` out - typically stuff like this would be a code smell if trying to go the OOP route. It's effectively the same, but keeping this on `BaseController` and exposing a getter would allow access to this.

I think part of the danger with it being public vs using a getter is that if it's public, it could be changed by something else, where using a getter I think it would not be the case.